recipe_scrapers/bakels.py

```diff
@@ -0,0 +1,54 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+
+
+class Bakels(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "bakels.com.au"
+
+    def author(self):
+        return "Australian Bakels"
+
+    def title(self):
+        return self.soup.find("h1").get_text()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        div = self.soup.find("div", id="tab-ingredients_1")
+        if div:
+            ingredients = div.find_all(class_="text-xs-left")
+            array = []
+            flag = False
+            for ingredient in ingredients:
+                text = ingredient.get_text(strip=True)
+                if text == "Ingredient":
```

Comment:
> It might be possible to implement [ingredient group support](https://github.com/hhursev/recipe-scrapers/blob/57ac5ee9dc8dc14d53b68335dd765e5edfcd4c52/docs/in-depth-guide-ingredient-groups.md) for this site, although I'm going to experiment with the code here to figure out whether that's more difficult than I think.

Hrm; this _does_ seem more difficult than I thought.  In particular I don't think that our standard `group_ingredients` helper function is able to handle this website easily (although I would be glad to be proven wrong about that).

@heathrampazis Since I did experiment with this and get some results working, I've opened a pull request at https://github.com/heathrampazis/recipe-scrapers/pull/1 with those changes for your repository.  Please note: I'd generally recommend using a branch other than `main` when developing functionality intended to be sent to other `git` repositories.

Reply:
I've had a look at your PR and it appears that you were able to get the grouping situation working as far as I can see. What specifically were you not able to get working, and if its all good, I'll submit a new commit with the changes I've made from your review.

PS: No worries, for future contributions I will make sure to use seperate branches. This is good programming practice 👍

Reply:
Thanks @heathrampazis.  The problem that I encountered (and worked-round in that pull request) was that I couldn't seem to get the [`group_ingredients()` function](https://github.com/hhursev/recipe-scrapers/blob/9fa183e27427d9745fc9e99d7340d929f265247d/docs/in-depth-guide-ingredient-groups.md#_grouping_utilspy) from this library to simplify the code.

When it _is_ possible to use that function, it makes the code fairly concise - here's a link to an example of that: https://github.com/hhursev/recipe-scrapers/blob/9fa183e27427d9745fc9e99d7340d929f265247d/recipe_scrapers/nhshealthierfamilies.py#L66-L72

So it's not a problem for the code here, but it could be a useful utility to know about.

Reply:
Ahh yep, I'm having a look now but am personally having no luck as well.

Reply:
Update - I think I got it to work using `group_ingredients()`. It seems to pass the tests as well, and I have been able to get it to return `null` when there is only 1 group. I will submit another commit now 🎉